packages/uikit/src/atoms/Button/Button.test.tsx

```diff
@@ -67,31 +63,34 @@ describe('Buttons', () => {
     });
   });
 
-  describe('Secondary Button', () => {
+  describe('Outline Button', () => {
     it('renders correctly', () => {
-      expect(setup(SecondaryButton)).toMatchSnapshot();
+      expect(setup(OutlineButton)).toMatchSnapshot();
     });
 
-    it('renders correctly in disabled state', () => {
-      expect(setup(SecondaryButton, true)).toMatchSnapshot();
-    });
-
-    it('renders correctly in loading state', () => {
-      expect(setup(SecondaryButton, false, true)).toMatchSnapshot();
-    });
-  });
-
-  describe('Tertiary Button', () => {
-    it('renders correctly', () => {
-      expect(setup(TertiaryButton)).toMatchSnapshot();
+    it('renders white outline Button correctly', () => {
+      const wrapper = setup(OutlineButton, false, false, 'primary');
+      expect(wrapper.find(OutlineButton).prop('colour')).toBe('primary');
+      expect(wrapper).toHaveStyleRule('color', theme.colours.primary);
+      expect(wrapper).toHaveStyleRule('background', theme.colours.whiteDenim);
+      expect(wrapper).toHaveStyleRule(
+        'border',
+        `1px solid ${theme.colours.primary}`
+      );
     });
 
     it('renders correctly in disabled state', () => {
-      expect(setup(TertiaryButton, true)).toMatchSnapshot();
+      expect(setup(OutlineButton, true)).toMatchSnapshot();
```

Comment:
Think that this expectation should be in its own describe block (e.g. `describe('When loading', () => {})`